beacon-chain/cache/depositcache/deposits_cache.go

```diff
@@ -171,3 +240,36 @@ func (dc *DepositCache) DepositByPubkey(ctx context.Context, pubKey []byte) (*et
 	}
 	return deposit, blockNum
 }
+
+// FinalizedDeposits returns the list of finalized deposits.
```

Comment:
Not exactly accurate, this just returns the finalized deposit trie. The deposits are not stored anywhere in the object